pandas/_libs/tslibs/parsing.pyx

```diff
@@ -63,6 +63,7 @@ from pandas._libs.tslibs.util cimport (
     get_c_string_buf_and_size,
     is_array,
 )
+from pandas.util._exceptions import find_stack_level
```

Comment:
@jbrockmendel any suggestions on how to do this without importing from outisde `_libs` (as you'd previously requested on a different PR)?

Reply:
you could import it when u need this (eg near usage)

or move the implementation to _libs itself

Reply:
I think this is a case where the rule-of-thumb preventing non-_libs imports is OK to break